pages/profile/read/index.js

```diff
@@ -6,10 +6,5 @@ module.exports = settings => {
     root: __dirname
   });
 
-  app.get('/', (req, res, next) => {
-    res.locals.static.isUser = req.user.profile.id === req.profileId;
```

Comment:
Is this resolved elsewhere?

Reply:
I can't see anywhere that `isUser` gets used when viewing a profile? 

e.g. https://github.com/UKHomeOffice/asl-pages/blob/master/pages/profile/read/views/index.jsx doesn't use it

Reply:
we set `isOwnProfile` in the other PR:
https://github.com/UKHomeOffice/asl/pull/173/files

Reply:
It's set on the dashboard where it's always true, but not set on the profile route where it might not be true.

I think this middleware needs to stay.